internal/Picker/Picker.js

```diff
@@ -332,11 +403,21 @@ const PickerBase = kind({
 						aria-label={decrementAriaLabel}
 						className={css.secondaryItemDecrement}
 						disabled={isSecond}
-						onClick={handleSecondaryDecrement}
+						onClick={() => {
+							handleDecrement(); setTimeout(() => handleDecrement(), 150);
+						}}
 					>
-						<div className={css.label}>
-							{isSecond ? '' : secondaryDecrementValue}
-						</div>
+						<ViewManager
+							aria-hidden
+							arranger={arranger}
+							className={css.viewManager}
+							duration={150}
```

Comment:
Let's define `150` as a constant variable so that we could tune is easily later.